extensions/blocks/anchor-fm/editor.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Internal dependencies
+ */
+import { name } from '.';
+import getJetpackExtensionAvailability from '../../shared/get-jetpack-extension-availability';
+import { dispatch } from '@wordpress/data';
+import { createBlock } from '@wordpress/blocks';
+
+/*
+ * Register the main "anchor-fm" extension.
+ */
+const isExtensionAvailable = getJetpackExtensionAvailability( name )?.available;
+if ( isExtensionAvailable ) {
+	// Load data from the backend.
+	const data = window.Jetpack_AnchorFm;
+	if ( typeof data === 'object' ) {
+		// Insert the link badge if needed.
+		const { track, podcast_id, episode_id } = data;
+		if ( typeof track === 'object' && podcast_id && episode_id ) {
+			window.onload = () => {
+				// Insert badge.
+				dispatch( 'core/block-editor' ).insertBlocks(
+					[
+						createBlock( 'core/image', {
+							url: 'https://cldup.com/Dv6JZWyRpq-1200x1200.png',
+							linkDestination: 'none',
+							href: track.link,
```

Comment:
This link seems to be an Anchor URL, not the Spotify show URL.

Example: https://anchor.fm/s/3e51070/podcast/rss

```
<link>https://anchor.fm/marekhrabe-testing/episodes/My-first-episode-elah21</link>
```

The Spotify show URL will be a new parameter that is passed by Anchor so we won't need to fetch any data from the podcast RSS:

- 346-gh-Automattic/dotcom-manage
- D53608-code

Reply:
9ef3a35